Intel_SighPP_CMake/CMakeLists.txt

```diff
@@ -3,28 +3,19 @@
 #
 cmake_minimum_required (VERSION 3.8)
 
-set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib)
-set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin)
-#set(CMAKE_LIBRARY_OUTPUT_DIRECTORY "${CMAKE_BINARY_DIR}/<$<$CONFIG:DEBUG$>:${OUTPUT_DEBUG}$><$<$CONFIG:RELEASE$>:${OUTPUT_REL}$>")
-#set(CMAKE_LIBRARY_OUTPUT_DIRECTORY_DEBUG ${CMAKE_BINARY_DIR}/${OUTPUT_DEBUG})
-# Output directory for libraries in Release configuration
-#sset(CMAKE_LIBRARY_OUTPUT_DIRECTORY_RELEASE ${CMAKE_BINARY_DIR}/${OUTPUT_REL})
-#set(CMAKE_TOOLCHAIN_FILE C:/Users/svens/vcpkg/scripts/buildsystems/vcpkg.cmake
-# CACHE STRING "Vcpkg toolchain file")
-#set(VCPKG_TARGET_TRIPLET x64-windows)
-
-#set (CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR})
 project (Intel_SighPP_CMake VERSION 0.1)
-find_package(OpenCV 3.4 CONFIG REQUIRED)
-find_package(realsense2 CONFIG REQUIRED)
-find_package(TBB CONFIG REQUIRED)
-find_package(websocketpp CONFIG REQUIRED)
-find_package(GTest CONFIG REQUIRED)
-find_package(spdlog 1.6 CONFIG REQUIRED)
 
-add_subdirectory ("Intel_SighPP_CMake")
+# Avoids "Undefined symbols for architecture" Error
+add_definitions(-DFMT_HEADER_ONLY)
 
-#find_package(GTest CONFIG REQUIRED PATHS C:/Users/svens/vcpkg/installed/x64-windows/share/gtest NO_DEFAULT_PATH)
-#find_package(OpenCV CONFIG REQUIRED PATHS C:/Users/svens/vcpkg/installed/x64-windows/share/opencv3 NO_DEFAULT_PATH)
-#find_package(realsense2 CONFIG REQUIRED PATHS C:/Users/svens/vcpkg/installed/x64-windows/share/realsense2 NO_DEFAULT_PATH)
-#find_package(TBB CONFIG REQUIRED PATHS C:/Users/svens/vcpkg/installed/x64-windows/share/tbb NO_DEFAULT_PATH)
\ No newline at end of file
+set (CMAKE_CXX_STANDARD 11)
+
+# Include sub-projects.
+find_package(Protobuf REQUIRED PATHS /Users/xingdacen/Downloads/vcpkg/installed/x64-osx/share/protobuf NO_DEFAULT_PATH)
```

Comment:
Did you mean to add these? 😄

Reply:
This is just the CMakeLists.txt I used for running the system.
Will replace this one with the original one from the Development branch.